ts/store/reducers/exitApp.ts

```diff
@@ -0,0 +1,43 @@
+import { getType } from "typesafe-actions";
+
+import { none, Option, some } from "fp-ts/lib/Option";
+import { Second } from "italia-ts-commons/lib/units";
+import {
+  navigationHistoryEmpty,
+  navigationHistoryPush
+} from "../actions/navigationHistory";
+import { Action } from "../actions/types";
+
+export type ExitAppState = Readonly<{
+  exitRequestTime: Option<Second>;
+}>;
+
+const INITIAL_STATE: ExitAppState = {
+  exitRequestTime: none
+};
+
+export function exitAppReducer(
+  state: ExitAppState = INITIAL_STATE,
+  action: Action
+): ExitAppState {
+  switch (action.type) {
+    // when the history is empty the app is eligible to exit
+    // exitRequestTime contains the current time value in seconds
+    case getType(navigationHistoryEmpty): {
+      const now = (new Date().getTime() / 1000) as Second;
```

Comment:
I guess we can just use millis and avoid the division by 1000 ?

Reply:
Sure! done